configure.ac

```diff
@@ -174,7 +173,7 @@ PKG_CHECK_MODULES(DBUS,[
  dbus-1 >= $DBUS_REQUIRED
  gthread-2.0])
 
-PKG_CHECK_MODULES(MATE, [
+PKG_CHECK_MODULES(CAIRO, [
```

Comment:
yeah, i noticed that there is a propblem if i use both current PRs on top of master with using autoreconf.
I will rework all when other PR is merged.